doc/private_net_sync.rst

```diff
@@ -0,0 +1,306 @@
+Creating A Private Network With Two Syncing Nodes
+==================================================
+These instructions cover creating an Ethereum private network consisting of two nodes - one node will mine blocks and the other node will connect to the first node and sync the mined blocks to its own block database.
+
+
+What is a Private Network?
+--------------------------
+For the purposes of this documentation, a **private network** can be thought of as a network of Ethereum nodes only accessible on your physical machine.
+
+What is a Private Chain?
+-------------------------
+An Ethereum chain is some state (e.g. accounts and balances and/or contract code) and a set of rules for interacting with that state. For the purposes of this documentation, a **private chain** is an Ethereum chain whose configuration is only available on your physical machine.
+
+Mining
+------
+- Mining will be done using the Aleth CPU miner (Aleth doesn't include a GPU miner because of the high maintenance and support costs. Please see `EthMiner <https://github.com/ethereum-mining/ethminer>`_ for GPU mining support).
+- We will only start mining for one node to keep your machine responsive.
+- If mining slows down your system too much or you'd like to have both nodes mine, you can tune the number of mining threads via ``-t <thread count>``.
+
+Chain Configuration
+-------------------
+- You typically initialize a private chain using a chain configuration json file (this isn't strictly required, but it makes testing a lot easier since you can do things like lower the difficulty rate and pre-fund addresses).
+- The chain configuration json file format is defined here: https://github.com/ethereum/wiki/wiki/Ethereum-Chain-Spec-Format
```

Comment:
Do we really follow that spec? It might be outdated.

Reply:
Good question! I assumed we did, Ill investigate

Reply:
You're right, the spec is either deprecated or we don't fully implement the spec (for example it lists a "nodes" key which we don't support). I'll remove the link.